pkg/chartutil/save_test.go

```diff
@@ -33,37 +34,39 @@ func TestSave(t *testing.T) {
 	}
 	defer os.RemoveAll(tmp)
 
-	c := &chart.Chart{
-		Metadata: &chart.Metadata{
-			APIVersion: chart.APIVersionV1,
-			Name:       "ahab",
-			Version:    "1.2.3",
-		},
-		Files: []*chart.File{
-			{Name: "scheherazade/shahryar.txt", Data: []byte("1,001 Nights")},
-		},
-	}
-
-	where, err := Save(c, tmp)
-	if err != nil {
-		t.Fatalf("Failed to save: %s", err)
-	}
-	if !strings.HasPrefix(where, tmp) {
-		t.Fatalf("Expected %q to start with %q", where, tmp)
-	}
-	if !strings.HasSuffix(where, ".tgz") {
-		t.Fatalf("Expected %q to end with .tgz", where)
-	}
-
-	c2, err := loader.LoadFile(where)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if c2.Name() != c.Name() {
-		t.Fatalf("Expected chart archive to have %q, got %q", c.Name(), c2.Name())
-	}
-	if len(c2.Files) != 1 || c2.Files[0].Name != "scheherazade/shahryar.txt" {
-		t.Fatal("Files data did not match")
+	for _, dest := range []string{tmp, path.Join(tmp, "newdir")} {
```

Comment:
Would you mind changing this to a `t.Run`? That way, these two tests cases can run in parallel. https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks

Reply:
@bacongobbler Changed it. Thank you!